src/components/QuickStart/QuickStart.tsx

```diff
@@ -60,6 +60,7 @@ export default function QuickStart() {
         isExpanded={isExpanded}
         displaySize='large'
         aria-label='quickStart-expansion'
+        data-ouia-component-id='quickstart_expand'
```

Comment:
Should be `ouiaId`?

Reply:
Also, does ExpandableSection support ouiaId ? 🤔 I can't seem to find it in patternfly. Guess I am looking at wrong location ?

Reply:
So, yeah it's `data-ouia-component-id` only.

Because this component does not support `ouiaID` variable so am directly using the value.

cc: @Andrewgdewar